apis/supernova/src/handler/utils/hypercube-helper/remove-alternative-dimension.js

```diff
@@ -0,0 +1,15 @@
+function removeAlternativeDimension(self, index) {
+  const [dimension] = self.getAlternativeDimensions().splice(index, 1);
+  if (typeof self.dimensionDefinition.remove === 'function') {
```

Comment:
I think it was better before when you checked `dimension` for truthness. Since `splice` can return an empty array.